### PR TITLE
add automation test for empty annotation

### PR DESCRIPTION
**Pull request by Avi-Eyal**
Added automation sanity test for empty annotation

### CODE REVIEW

File: cucumber/authenticators_jwt/features/authn_jwt.feature [hunk 1]
```diff
@@ -104,3 +104,29 @@ Feature: JWT Authenticator - JWKs Basic sanity
...
     """
+
+  Scenario: Annotation with empty value
+    Given I have a "variable" resource called "test-variable"
+    Given I extend the policy with:
+    """
+    - !host
+      id: myapp
+      annotations:
+        authn-jwt/raw/custom-claim:
```
Comment (tzheleznyak) [inline cucumber/authenticators_jwt/features/authn_jwt.feature:115]: Add more annotations here so only one of them with empty value

Comment (Avi-Eyal) [reply cucumber/authenticators_jwt/features/authn_jwt.feature:115]: there are already other annotations in the policy that is defined in the background

Comment (Avi-Eyal) [reply cucumber/authenticators_jwt/features/authn_jwt.feature:115]: they are defined in the same host (`myapp`) so i dont see why we need to add another annotation

and about the files for each test, Aviram and I talked about it and we thought that it would be better to wait until we finish to write the tests and only then we will split the tests to different files

**Review by tzheleznyak — APPROVED** [review]
